### PR TITLE
Tags example

### PR DESCRIPTION
**Pull request by lstadnik**
## Description

Create a regional VMseries deployment example with the network instance tags.

## Motivation and Context

#77 

## How Has This Been Tested?

-VMseries bootstrap status
-Traffic outbound from Instances

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

File: modules/vpc/main.tf [hunk 1]
```diff
@@ -65,7 +65,7 @@ resource "google_compute_subnetwork" "this" {
 }
 
 resource "google_compute_firewall" "this" {
-  for_each      = { for k, v in local.networks : k => v if can(v.allowed_sources) }
+  for_each      = { for k, v in local.networks_to_create : k => v if can(v.allowed_sources) }
```
[the remark this answers is not in the document]
Comment (lstadnik) [reply modules/vpc/main.tf:68]: No It was a bug - the existing network should not be created

File: examples/common_firewall_with_networks_tags/bootstrap_files/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Manual operation
```
[the remark this answers is not in the document]
Comment (lstadnik) [reply examples/common_firewall_with_networks_tags/bootstrap_files/README.md:1]: let's look at common firewall example, we can update it later, globally.

File: examples/common_firewall_with_networks_tags/example.tfvars [hunk 3]
```diff
@@ -0,0 +1,70 @@
+project_id      = "gcp-gcs-pso"
+project         = "gcp-gcs-pso"
+region          = "us-central1"
+region0         = "us-central1"
```
[the remark this answers is not in the document]
Comment (lstadnik) [reply examples/common_firewall_with_networks_tags/example.tfvars:3]: too much effort in compare to benefit in this example

**Review by michalbil — APPROVED** [review]

File: modules/vpc/main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (lstadnik) [reply modules/vpc/main.tf]: leave as it is

**Pull request merged**